### PR TITLE
deletes transaction

### PR DESCRIPTION
**Pull request by srilaxmi14**

### CODE REVIEW

File: src/FileCabinet/SuiteScripts/undo_transaction_ue.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+               ["type","anyof","ItemShip"], 
+               "AND", 
+               ["mainline","is","T"]
+            ],
+            columns:
+            [
+               search.createColumn({name: "internalid", label: "Internal ID"})
+            ]
+         });
+         var itemFulfillments = itemfulfillmentSearch.run().getRange(0, 10);
```
Comment (jampalu) [inline src/FileCabinet/SuiteScripts/undo_transaction_ue.js:42]: use getRange(0,100), just to be on the safer side.
can be more but minimum should be 100

File: src/FileCabinet/SuiteScripts/undo_transaction_ue.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            ]
+         });
+         var invoices = invoiceSearch.run().getRange(0, 10);
+
+         
+         if (itemFulfillments.length > 0 || invoices.length > 0) {
+             scriptContext.form.addButton({
+               id: "custpage_undo_button",
+               label: "Undo",
+               functionName: 'deletetransaction(' + salesOrderId + ')',
```
Comment (jampalu) [inline src/FileCabinet/SuiteScripts/undo_transaction_ue.js:67]: Don't forget to use camelCase for all the variables, and functions.
change your function name to "deleteTransactions", should be plural as we are deleting more than one.

File: src/FileCabinet/SuiteScripts/undo_transaction_cs.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            ["type", "anyof", "ItemShip"],
+            "AND",
+            ["mainline", "is", "T"]
+          ],
+        columns:
+          [
+            search.createColumn({ name: "internalid", label: "Internal ID" })
+          ]
+      });
+      var itemFulfillments = itemfulfillmentSearch.run().getRange(0, 10);
```
Comment (jampalu) [inline src/FileCabinet/SuiteScripts/undo_transaction_cs.js:36]: update range here as well to getRange(0,100)

**Review by jampalu — CHANGES_REQUESTED** [review]

File: src/FileCabinet/SuiteScripts/undo_transaction_ue.js (hunk 1 above)
Comment (srilaxmi14) [reply src/FileCabinet/SuiteScripts/undo_transaction_ue.js]: Done with changes.

File: src/FileCabinet/SuiteScripts/undo_transaction_ue.js (hunk 2 above)
Comment (srilaxmi14) [reply src/FileCabinet/SuiteScripts/undo_transaction_ue.js]: Done with changes.

File: src/FileCabinet/SuiteScripts/undo_transaction_cs.js (hunk 3 above)
Comment (srilaxmi14) [reply src/FileCabinet/SuiteScripts/undo_transaction_cs.js]: Done with changes.

**Pull request merged**